src/runtime/auto-prompt-manager.ts

```diff
@@ -224,15 +224,14 @@ export class AutoPromptManager {
     if (!article) {
       return;
     }
+    const articleExpFlags =
+      this.entitlementsManager_.parseArticleExperimentConfigFlags(article);
 
-    this.frequencyCappingLocalStorageEnabled_ =
-      this.isArticleExperimentEnabled_(
-        article,
-        ArticleExperimentFlags.FREQUENCY_CAPPING_LOCAL_STORAGE
-      );
+    this.frequencyCappingLocalStorageEnabled_ = articleExpFlags.includes(
+      ArticleExperimentFlags.FREQUENCY_CAPPING_LOCAL_STORAGE
+    );
 
-    this.promptFrequencyCappingEnabled_ = this.isArticleExperimentEnabled_(
-      article,
+    this.promptFrequencyCappingEnabled_ = articleExpFlags.includes(
       ArticleExperimentFlags.PROMPT_FREQUENCY_CAPPING_EXPERIMENT
     );
```

Comment:
Can this use the new method you added in EntitlementsManager instead of re-implementing the flags.includes check?